src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Shapes/Shapes_Tests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using SamplesApp.UITests.TestFramework;
+
+namespace SamplesApp.UITests.Windows_UI_Xaml_Shapes
+{
+	public class Shapes_Tests : SampleControlUITestBase
+	{
+		[Test]
+		[AutoRetry]
+		public void Draw_polyline()
+		{
+			Run("SamplesApp.Windows_UI_Xaml_Shapes.PolylinePage");
+			_app.WaitForElement("DPolyline");
+			_app.Screenshot($"PolylinePage");
+		}
+
+		[Test]
+		[AutoRetry]
+		public void Draw_polygon()
+		{
+			Run("SamplesApp.Windows_UI_Xaml_Shapes.PolygonPage");
+			_app.WaitForElement("DPolygon");
+			_app.Screenshot($"PolygonPage");
+		}
+
+		[Test]
+		[AutoRetry]
+		public void Draw_ellipse()
+		{
+			Run("SamplesApp.Windows_UI_Xaml_Shapes.EllipsePage");
+			_app.WaitForElement("DEllipsePage");
+			_app.Screenshot($"EllipsePage");
+		}
+
+		[Test]
+		[AutoRetry]
+		public void Draw_line()
+		{
+			Run("SamplesApp.Windows_UI_Xaml_Shapes.LinePage");
+			_app.WaitForElement("DLinePage");
+			_app.Screenshot($"LinePage");
```

Comment:
You don't need to create a unit test to take a screenshot... there's already a task taking a screenshot of all samples.